.github/workflows/ios.yml

```diff
@@ -0,0 +1,37 @@
+name: iOS CI
+
+on:
+  push:
+    branches: [ master]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build and Test default scheme using any available iPhone simulator
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Relisten
+        run: |
+          git clone --quiet https://github.com/alecgorge/gapless-audio-bass-ios.git ../BASSGaplessAudioPlayer
+          git clone --quiet https://github.com/alecgorge/AGAudioPlayer.git ../AGAudioPlayer
+          pod update RealmSwift
+          ./setup.sh
```

Comment:
This script should do the git clones that you have above. It looks like both Travis and GitHub Actions define the `CI` environment variable, so we should switch the script to checking for that. We can also change the path in `setup.py`, `Podfile`, and `.gitignore` to be something more generic like `CIPods`

Reply:
I think we may also need to change `setup.py` to run `bundle exec pod` instead of just `pod` everywhere. That should be safe to run on developer machines too.